Azure.ResourceManager.Core.Tests/Resource/TaggableResource.cs

```diff
@@ -54,6 +54,41 @@ public override void Validate(ResourceIdentifier identifier)
         {
         }
 
+        /// <summary>
```

Comment:
Sync master this file no longer exists.  I had to update these tests so they don't have hard coded resources in them so it runs for everyone.